src/index.html

```diff
@@ -1,14 +1,89 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- я знаю, що не попадаю по пікселям,
+      але вже година ночі - пощади
+     -->
```

Comment:
Sorry 😢

Reply:
Problem is not just in pixels, there were few other issues that need to bi fixed and I believe that this will help to fix some of tests. It's okay if some of tests will be broken because of pixel-perfect, but other issues should be fixed